packages/bvaughn-architecture-demo/src/contexts/ConsoleFiltersContext.tsx

```diff
@@ -59,18 +58,30 @@ export type ConsoleFiltersContextType = Toggles & {
   ) => void;
 };
 
+// NOTE: If any change is made like adding a store name, bump the version number
+// to ensure that the database is recreated properly.
+export const CONSOLE_SETTINGS_DATABASE: IDBOptions = {
```

Comment:
nit Since this isn't filter-specific but console-wide, maybe this constant should live somewhere else? Don't feel strongly about it.

Reply:
Yeah, I wasn't sure whether to put this in its own file or not - it's related to the rest of the console logic, but wasn't strictly a "component", so I figured this was the closest place to put it. I can move it around.